change/@microsoft-fast-animation-32d3c5b6-562a-4962-8297-e95d92c3f294.json

```diff
@@ -1,7 +1,7 @@
 {
-  "type": "patch",
+  "type": "prerelease",
```

Comment:
I *think* this one should be okay to leave as 'patch' if none of the pre-release packages depend on it

Reply:
Ah! I can change that back. I've had a practice of checking all the things when I commit and fixing them, just in case.